openstudiocore/src/utilities/filetypes/EpwFile.cpp

```diff
@@ -132,67 +132,119 @@ EpwDataPoint::EpwDataPoint(int year,int month,int day,int hour,int minute,
   setLiquidPrecipitationQuantity(liquidPrecipitationQuantity);
 }
 
-boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(std::string line)
+boost::optional<EpwDataPoint> EpwDataPoint::fromEpwString(const std::string &line)
+{
+  std::vector<std::string> list;
+  boost::split(list, line, boost::is_any_of(","));

```

Comment:
Interesting, new method I didn't know about.  We do want to be careful not to introduce any new boost dependencies though.  QString might be preferred to this, unless C++11 has some spiffy method?